### PR TITLE
SONAR-11714 Remove .badge from Select list CSS

### PR DESCRIPTION
**Pull request by wouter-admiraal-sonarsource**
Ahem, sorry... Should have linked locally... The new design doesn't work well with these styles.

Relates to https://github.com/SonarSource/sonar-enterprise/pull/1930

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,9 @@ language: node_js
 sudo: false
 install: true
 jdk: oraclejdk11
-node_js: "10.16.0"
+node_js: "8"
```
Comment (gregaubert) [inline .travis.yml]: To align with `sonar-enterprise` it should be `10.15.3`, not `8`

File: .travis.yml [hunk 2]
```diff
@@ -2,11 +2,9 @@ language: node_js
...
 dist: trusty
 
-before_install: if [[ `yarn -v` != 1.17* ]]; then curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.17.3 && export PATH="$HOME/.yarn/bin:$PATH"; fi
```
Comment (gregaubert) [inline .travis.yml:8]: I don't know what the default version is on Travis, let's use `1.15.2` to align with `sonar-enterprise`.
Or update `build.gradle` from `sonar-enterprise` to use a more recent one.

File: .travis.yml [hunk 3]
```diff
@@ -2,10 +2,10 @@ language: node_js
 sudo: false
 install: true
 jdk: oraclejdk11
-node_js: "10.16.0"
+node_js: "10.15.3"
```
Comment (siegfried-ehret-sonarsource) [inline .travis.yml]: @wouter-admiraal-sonarsource 10.16.0 (current LTS) is not available on Travis ?

Comment (wouter-admiraal-sonarsource) [reply .travis.yml]: We align it on sonar-enterprise, which uses 10.15.3.

Comment (siegfried-ehret-sonarsource) [reply .travis.yml]: Shouldn't we align the other way and have the latest LTS ?

Comment (wouter-admiraal-sonarsource) [reply .travis.yml]: We could, although we don't care that much (IMHO). It's only for building, not running any application itself. And I realize we're speaking of the `.travis.yml` file, which yes, we can def. set to the LTS 😅 . But we can keep 10.15.3 in `package.json`.

**Review by gregaubert — APPROVED** [review]

**Pull request merged**